__tests__/test.tjbotlib.js

```diff
@@ -15,7 +15,27 @@
  * limitations under the License.
  */
 
-import TJBot from '../lib/tjbot.js';
+// import { config } from 'bluebird';
```

Comment:
is it necessary to keep unused imports? :)

Reply:
Fixed!